app/src/components/Header/AuthHeader.jsx

```diff
@@ -8,7 +8,7 @@ const Header = () => {
     <header className="authed">
       <img src={logo} alt="logo" />
       <h1>FFS Chat</h1>
-      <Nav />
+      <NavMenu />
```

Comment:
This could more appropriately be named AppMenu I think, and then when the chat menu gets introduced we can call it ChatMenu to differentiate, since it isn't *really* a nav

Reply:
Sure, makes sense. I'll update